games-puzzle/nudoku/metadata.xml

```diff
@@ -1,7 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<!-- maintainer-needed -->
+		<maintainer type="person" proxied="yes">
```

Comment:
align indetation of this block, one tab left.

Reply:
My bad! I changed that and pushed. but seems already merged?

Reply:
I fixed those while merging it. But certainly something you may want to look at in the future. :)

Reply:
Will do! Thanks